pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -780,14 +780,14 @@ public void start() throws PulsarServerException {
 
             acquireSLANamespace();
 
-            // start function worker service if necessary
-            this.startWorkerService(brokerService.getAuthenticationService(), brokerService.getAuthorizationService());
-
             // start packages management service if necessary
             if (config.isEnablePackagesManagement()) {
                 this.startPackagesManagementService();
             }
```

Comment:
If you make function worker depend on package management service, then shouldn't the package management service be always enabled when function worker is enabled?

Currently it seems they are separately configured, which may result the case that function worker is enabled but package management service is not running.

Reply:
There are two situations:
1、Run Functions-worker with brokers. If broker enables  function worker without enabling packageManagement,  packageManagement service is enabled in worker by default. Else, worker will use packageManagement service enabled by broker.
2、Run Functions-worker separately. Without broker, We enable packageManagement service  in worker by default